CMakeLists.txt

```diff
@@ -57,3 +73,62 @@ if(MSVC_VERSION GREATER_EQUAL 1936 AND MSVC_IDE) # 17.6+
 </Project>
 ]==] @ONLY)
 endif()
+
+# #######################################################################################################################
+# # Automatic deployment
+# #######################################################################################################################
+
+file(GLOB FEATURE_PATHS LIST_DIRECTORIES true ${CMAKE_SOURCE_DIR}/features/*)
+
+# Automatic deployment to CommunityShaders output directory.
+if (AUTO_PLUGIN_DEPLOYMENT)
+	if (DEFINED ENV{CommunityShadersOutputDir})
```

Comment:
Can we make it a for each so multiple outputs can be defined? I test in flat and VR so need to update both.

Reply:
Fixed! Also added a CMakeUserPresets.json.template how to use it with multiple